src/Component/Button.js

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+
+function Button {
+    function Button({ clickHandler, children, type, disabled }) {
+        return (
+            <button
+                onClick={clickHandler}
+                type={type}
+                disabled={disabled || false}
+            >
+            </button>
+        );
+    }
+}
```

Comment:
Dit component zorgt niet voor een vermindering in het aantal regels code. De hoeveelheid code die nodig is om dit component aan te roepen is net zo groot als de inhoud van dit component. Hier zou ik dus geen los component van maken